### ISSUE TITLE
Dashboard not work

### ISSUE THREAD

**Issue by ImpMaple51** [SudhanPlayz/Discord-MusicBot#762]
Dashboard always invalid oauth2 even though I've added a redirect link/api/callback

**Comment by DarrenOfficial** [comment]
not a bug, most likely you've fu**ked up the config
Example of the correct configuration
![](https://cdn.darrennathanael.com/assets/discord/oauth2dis.png)

**Comment by rolkoztm** [comment]
@DarrenOfficial no, look at my pull req it will fix it

**Comment by DarrenOfficial** [comment]
No, your pull request is invalid first of all, URI is correct. It's not URL.
Secondly this person just had an Invalid config, all he needs is to add the "Redirets" at the Discord Oauth2 Page
